packages/mobile/src/screens/chat-screen/ChatBlastHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import { useChatBlastAudienceContent } from '@audius/common/hooks'
+import type { ChatBlast } from '@audius/sdk'
+
+import { Flex, Text, IconTowerBroadcast } from '@audius/harmony-native'
+
+const messages = {
+  chatBlastTitleAudienceCount: (audienceCount: number) => `(${audienceCount})`
```

Comment:
lol maybe overkill idk

I guess this could be different in different locales?

Reply:
lol idk... it's either this or just put parents in the string directly..